README.md

```diff
@@ -113,3 +113,30 @@ npm run watch
 ```
 npm test
 ```
+
+## Upgrading Chromedriver Version
+
+When a new [Chromedriver](http://chromedriver.chromium.org/) version is released,
+the details will be [here](http://chromedriver.chromium.org/downloads). Which
+Chromedriver this package selects is based on the `CHROMEDRIVER_CHROME_MAPPING`
+in `lib/chromedriver`. Add a new entry to the top, with the correct version number
+and a random, but low, "minimum chrome version". To install, build then re-install
```

Comment:
> a random, but low, "minimum chrome version".

Is it by design? I'd rather just use some magic char there, like `_` to reflect, that we don't care about such number

Reply:
This is just so that it fails in the test step. Then the real number is added. This never gets past the running of a single test.

Reply:
should we either ask chromedriver devs to add an explicit command line argument, which is going to show the minimum supported version instead of doing such ugly hacks?

Reply:
Reworded things. Hopefully clearer now.

Reply:
I did ask them. Nothing ever came of it.

Reply:
In terms of "hacks" this is minimal. It never is used by anyone else... I just need to do this process when a new version comes out. And I didn't feel like remembering it anymore, so I wrote it down here.